lib/trino-filesystem-gcs/src/main/java/io/trino/filesystem/gcs/GcsOutputFile.java

```diff
@@ -69,7 +68,7 @@ public OutputStream createOrOverwrite(AggregatedMemoryContext memoryContext)
     }
 
     @Override
-    public void createExclusive(Slice content)
+    public void createExclusive(byte[] data)
```

Comment:
Consider adding a version with data, offset, and length, and a default for just data?  We can do this later also.

Reply:
None of the callers need this. We can add later if needed.